src/common/MoviesList/index.js

```diff
@@ -13,17 +17,21 @@ const MoviesList = ({ data }) => {
         ) : data.status === "error" ? (
           <StyledMainHeader>SOMETHING WENT WRONG, sorry.. </StyledMainHeader>
```

Comment:
I didn't really want to touch the MovieList since the other person is working on it, it could end up with additional unnecessary conflictsnI believe..

Reply:
Done 👌